### ISSUE TITLE
Doppelstudium

### ISSUE THREAD

**Issue by boredland** [MyGrades/mygrades-app#6]
Is there already support for it?

**Comment by tilmanginzel** [comment]
Hey! As I understand you want to select two separate universities, is that correct?

We haven't thought about that so far. Unfortunately, this would require some major changes, so I don't think it will be supported anytime soon. :\

**Comment by boredland** [comment]
No: one University and two study programs. Like IT and Maths for example.

Am 30.09.2016 1:42 nachm. schrieb "Tilman Ginzel" <notifications@github.com

**Comment by tilmanginzel** [comment]
Ah ok, that depends on the university and how they manage their system. The app relies on a structured list of grades with all neccessary information, e.g. a table. Most universities will provide this information in single table for both field of studies, but unfortunately not all do. This may even differ for a single university: some combinations are in a single table, some are not.

It is impossible for us to know, which university and which field of studies will appear in a single table, and which do not. So I'm afraid we cannot find a proper solution for that. :(

**Comment by boredland** [comment]
Closing this myself as the reporter — I'm no longer pursuing it, and I'd rather not leave it sitting in your triage queue.

No action is needed from the maintainers, and this says nothing about the validity of the report: if anyone else is hitting the same thing, please reopen or file fresh.

**Comment by boredland** [comment]
Reopening — please disregard my previous comment.

That closure was a mistake on my side: it came from a bulk sweep of my own filed issues, not from a considered decision to withdraw this report. Sorry for the noise in your notifications.

The report stands as originally written.
